cpp/tests/io/parquet_test.cpp

```diff
@@ -1285,10 +1264,9 @@ TEST_F(ParquetChunkedWriterTest, ReadRowGroups)
   auto filepath = temp_env->get_temp_filepath("ChunkedRowGroups.parquet");
   cudf_io::chunked_parquet_writer_options args =
     cudf_io::chunked_parquet_writer_options::builder(cudf_io::sink_info{filepath});
-  auto state = cudf_io::write_parquet_chunked_begin(args);
-  cudf_io::write_parquet_chunked(*table1, state);
-  cudf_io::write_parquet_chunked(*table2, state);
-  cudf_io::write_parquet_chunked_end(state);
+  {
+    cudf_io::parquet_chunked_writer(args).write(*table1).write(*table2);
+  }
```

Comment:
don't need the block here, the writer is a temporary.
```suggestion
  cudf_io::parquet_chunked_writer(args).write(*table1).write(*table2);
```